### PR TITLE
Fix all flake8 issues, run in CI

### PR DESCRIPTION
**Pull request by martinpitt**
Finally, ALE stops being a sea of red :grin:

### CODE REVIEW

File: src/systemd_ctypes/bus.py [hunk 1]
```diff
@@ -236,7 +236,7 @@ def _coroutine_task_complete(self, out_types: Sequence[str], task: asyncio.Task)
         except BusError as exc:
             self.reply_method_error(exc)
 
-    def reply_method_function_return_value(self, out_types: Sequence[str], return_value: Any) -> bool:  # Literal[True]:
```
Comment (allisonkarlitskaya) [inline src/systemd_ctypes/bus.py:239]: I wonder why I had this as a comment.  `Literal[True]` is clearly the correct thing to have here, not `bool`, but something was stopping that from working.

I'd prefer not to get rid of this completely without figuring it out first.

File: test/test_code.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import pytest
+import shutil
+import subprocess
+
+
+ROOT_DIR = os.path.realpath(f'{__file__}/../..')
+
+
+@pytest.mark.skipif(not shutil.which('flake8'), reason='flake8 is not installed')
+def test_flake8():
```
Comment (allisonkarlitskaya) [inline test/test_code.py:11]: <3

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]
Thanks for taking care of this!

Really just one gripe.  See below.

File: src/systemd_ctypes/bus.py (hunk 1 above)
Comment (martinpitt) [reply src/systemd_ctypes/bus.py:239]: OK, I put it back and split the arguments to separate lines instead.

**Review by allisonkarlitskaya — APPROVED** [review]

**Pull request merged**